core/src/main/java/com/linecorp/armeria/client/WebClient.java

```diff
@@ -330,6 +355,14 @@ default HttpResponse post(String path, HttpData content) {
         return execute(RequestHeaders.of(HttpMethod.POST, path), content);
     }
 
+    /**
+     * Sends an HTTP POST request with the specified content, accepts query parameters to append to the path.
+     */
+    @CheckReturnValue
+    default HttpResponse post(String path, HttpData content, QueryParams params) {
```

Comment:
Probably better to do 
```suggestion
    default HttpResponse post(String path, QueryParams params, HttpData content) {
```
I feel that the `path` should be followed by the `params` because they are used in that way in the URL. 😄

Reply:
Makes sense from a readability perspective, I've made the change for the overloaded methods of all verbs to include QueryParams after the path.

Reply:
Please bear with me if there are any more. 😄

Also, I've made quite a few changes since the last commit for which a build was triggered, could you please approve a build run on these changes? Just wanted to ensure we're good coverage-wise.